File: app/src/main/java/com/grass/helloandroid/KbApplication.java
package com.grass.helloandroid;

import android.app.Application;

import com.grass.helloandroid.epic.EpicInit;

/**
 * Created by grassswwang
 * on 2020/9/4
 * Email: grassswwang@tencent.com
 */
public class KbApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        new EpicInit().init();
    }
}

File: app/src/main/java/com/grass/helloandroid/KbApplication.kt
package com.grass.helloandroid

import android.app.Application

/**
 *
 * Created by grassswwang
 * on 2020/7/9
 * Email: grassswwang@tencent.com
 */
class KbApplication : Application() {

    companion object {
        var application: Application = Application();
    }

    override fun onCreate() {
        super.onCreate()
        application = this
    }
}